server/etcdserver/txn/txn_test.go

```diff
@@ -265,10 +264,10 @@ func setupAuth(t *testing.T, be backend.Backend) auth.AuthStore {
 
 	// create "root" user and "foo" user with limited range
 	if _, err := as.RoleAdd(&pb.AuthRoleAddRequest{Name: "root"}); err != nil {
-		t.Fatal(err)
+		require.NoError(t, err)
```

Comment:
It isn't correct.

Please change 
```
	if _, err := as.RoleAdd(&pb.AuthRoleAddRequest{Name: "root"}); err != nil {
		t.Fatal(err)
		require.NoError(t, err)
	}
```

to 

```
	err := as.RoleAdd(&pb.AuthRoleAddRequest{Name: "root"})
	require.NoError(t, err)
```

Reply:
Now I see, thanks.